### DISCUSSION TITLE
props type을 지정할 때 interface, type 선호하는 것

### DISCUSSION THREAD

**Discussion by kingyong9169** [Nexters/who-really-wants-to-play#19] [Q&A] [answered]
다들 혹시 props type을 지정할 때 interface, type 중에 선호하시는 것이 있을까요?
@1ilsang @ooooorobo @anyl92

**Comment by 1ilsang** [comment, marked as answer]
- https://velog.io/@lky5697/11-tips-that-help-you-become-a-better-typescript-programmer?utm_source=substack&utm_medium=email#7-interface%EB%B3%B4%EB%8B%A4-type%EC%9D%84-%EC%82%AC%EC%9A%A9

나는 주로 Interface로 props 작업했는데 위 글처럼 타입과 interface를 명확히 나눠서 모두 type으로 작업하는것도 나쁘지 않을거 같음.

**Reply by kingyong9169** [reply]
오 좋은 글이네요! 그럼 type으로 컨벤션 맞추면 어떨까요?

**Reply by 1ilsang** [reply]
gogo
